arangod/RestHandler/RestAdminLogHandler.cpp

```diff
@@ -472,3 +478,63 @@ void RestAdminLogHandler::handleLogLevel() {
                   TRI_ERROR_HTTP_METHOD_NOT_ALLOWED);
   }
 }
+
+void RestAdminLogHandler::handleLogStructuredParams() {
+  std::vector<std::string> const& suffixes = _request->suffixes();
+
+  // was validated earlier
+  TRI_ASSERT(!suffixes.empty());
+
+  if (suffixes[0] != "structured") {
+    generateError(rest::ResponseCode::BAD, TRI_ERROR_HTTP_SUPERFLUOUS_SUFFICES,
+                  "superfluous suffix, expecting /_admin/log/structured");
+    return;
+  }
+
+  auto const type = _request->requestType();
+
+  if (type == rest::RequestType::GET) {
+    VPackBuilder builder;
+    builder.openObject();
+    auto const params = Logger::structuredLogParams();
+    for (auto const& param : params) {
+      builder.add(param, VPackValue(true));
+    }
+    builder.close();
+
+    generateResult(rest::ResponseCode::OK, builder.slice());
+  } else if (type == rest::RequestType::PUT) {
+    // set log level
+    bool parseSuccess = false;
+    VPackSlice slice = this->parseVPackBody(parseSuccess);
+    if (!parseSuccess) {
+      return;  // error message generated in parseVPackBody
+    }
+
+    if (slice.isString()) {
+      Logger::setLogStructuredParams({slice.copyString()});
```

Comment:
Should the REST API actually be callable with a string input? I don't think this is required functionality.

Reply:
Removed it